src/Tribe/Attendees_Table.php

```diff
@@ -814,27 +814,78 @@ protected function attendee_reference( $reference ) {
 	public function prepare_items() {
 		$this->process_actions();
 
-		$event_id = isset( $_GET['event_id'] ) ? $_GET['event_id'] : 0;
+		$current_page = $this->get_pagenum();
+		$per_page     = $this->get_items_per_page( $this->per_page_option );
+
+		$pagination_args = [
+			'total_items' => 0,
+			'per_page'    => $per_page,
+		];
 
-		$items = Tribe__Tickets__Tickets::get_event_attendees( $event_id, true );
+		$args = [
+			'page'               => $current_page,
+			'per_page'           => $per_page,
+			'return_total_found' => true,
+		];
 
-		$search = isset( $_REQUEST['s'] ) ? esc_attr( trim( $_REQUEST['s'] ) ) : false;
-		if ( ! empty( $search ) ) {
-			$items = $this->filter_attendees_by_string( $search, $items );
+		$event_id = 0;
+
+		if ( ! empty( $_GET['event_id'] ) ) {
+			$event_id = absint( $_GET['event_id'] );
 		}
 
-		$total_items = count( $items );
+		$search = null;
 
-		$current_page = $this->get_pagenum();
-		$per_page = $this->get_items_per_page( $this->per_page_option );
-		$this->items  = array_slice( $items, ( $current_page - 1 ) * $per_page, $per_page );
+		if ( ! empty( $_REQUEST['s'] ) ) {
+			$search = sanitize_text_field( $_REQUEST['s'] );
```

Comment:
If you use a ternary here you don't have to nest the ifs:
`$search = ! empty( $_REQUEST['s'] ) ? sanitize_text_field( $_REQUEST['s'] ) : 0;`

Then you can pull out the `if ( ! empty( $search ) )`